presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -40,6 +40,18 @@ The following file types are supported for the Hive connector:
 * CSV (using ``org.apache.hadoop.hive.serde2.OpenCSVSerde``)
 * TextFile
 
+In order to enable first-class support for CSV files, you need to add the
+following property definition to the Hive metastore configuration file
```

Comment:
this is needed for Hive 3 only.

-- this is the only place where we set this in our tests 
https://github.com/prestosql/docker-images/blob/4aadfb19cf5a3d7a2576f4cd670c9ced21debf53/prestodev/hdp3.1-hive/files/etc/hive/conf/hive-site.xml#L69-L70

Reply:
@electrum, maybe we _also_ could catch exception somewhere around https://github.com/prestosql/presto/blob/ebe16716fb76e4c4883cf7196d9b1087edb51582/presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/BridgingHiveMetastore.java#L89 and provide more actionable message directly to the user?